x-pack/plugins/lists/public/exceptions/components/builder/exception_items_renderer.test.tsx

```diff
@@ -120,6 +120,96 @@ describe('ExceptionBuilderComponent', () => {
     );
   });
 
+  test('it displays "is in list" operators if "allowLargeValueLists" is true', async () => {
+    wrapper = mount(
+      <EuiThemeProvider>
+        <ExceptionBuilderComponent
+          allowLargeValueLists={true}
+          autocompleteService={autocompleteStartMock}
+          exceptionListItems={[
+            {
+              ...getExceptionListItemSchemaMock(),
+              entries: [
+                { ...getEntryMatchAnyMock(), field: getField('ip').name, value: ['some ip'] },
+              ],
+            },
+          ]}
+          httpService={mockKibanaHttpService}
+          indexPatterns={{
+            fields,
+            id: '1234',
+            title: 'logstash-*',
+          }}
+          isAndDisabled={false}
+          isNestedDisabled={false}
+          isOrDisabled={false}
+          listId="list_id"
+          listNamespaceType="single"
+          listType="detection"
+          ruleName="Test rule"
+          onChange={jest.fn()}
+        />
+      </EuiThemeProvider>
+    );
+
+    expect(
+      wrapper.find('[data-test-subj="operatorAutocompleteComboBox"]').at(0).prop('options')
+    ).toEqual([
+      { label: 'is' },
+      { label: 'is not' },
+      { label: 'is one of' },
+      { label: 'is not one of' },
+      { label: 'exists' },
+      { label: 'does not exist' },
+      { label: 'is in list' },
```

Comment:
Nit: IMO this test could be a little more robust and targeted if you used `toContain` here to check specifically for the value list operators. If another operator is added/removed in the future, this test will need to be updated even though the intent of the test (displaying value list operators when "allowLargeValueLists" is true) is unrelated.